h/presenters/annotation_searchindex.py

```diff
@@ -32,30 +32,18 @@ def asdict(self):
             "target": self.annotation.target,
             "document": docpresenter.asdict(),
             "thread_ids": self.annotation.thread_ids,
+            "hidden": self.annotation.is_hidden,
```

Comment:
In the annotation entity, `is_hidden` is a `@property` that is computed from the moderation status.